### ISSUE TITLE
Error message when two files define the same object should be improved

### ISSUE THREAD

**Issue by mkromberg** [Dyalog/link#169]
The error message is strangely aligned.
```
      ]link.create # c:\temp\linktest
* Command Execution Failed: ⎕SE.Link.U.UCMD_Run: Files produce clashing APL names: c:/temp/linktest/foo.dyalog  c:/temp/l-
                                                                                                                inktest/g-
                                                                                                                oo.dyalog
```
Windows v19.0 pre-release with Link 2.1.0-alpha

**Comment by mkromberg** [comment]
I suggest something like this:

      ]link.create # c:\temp\linktest
* Command Execution Failed: ⎕SE.Link.U.UCMD_Run: Files define the same APL name:
foo: c:/temp/linktest/foo.dyalog  c:/temp/linktest/goo.dyalog

**Comment by abrudz** [comment]
The user command framework understands the double space as indicating two columns. This will fix itself by ensuring there's only one. Don't do `⍕'abc' 'def'` but rather `∊' ',∘⍕¨'abc' 'def'`. Maybe even put double quotes around the names, just in case there are confusing spaces in the path.

**Comment by nicolas-dyalog** [comment]
Fixed in link v2.1-beta1
